apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -347,6 +349,73 @@ static void tc_libc_signal_psignal(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn                   :tc_libc_signal_sigwait
+* @brief                :tc_libc_signal_sigwait test sigwait function
+* @scenario             :If signo < 0, it return errno by get_errno().
+*                        Else if signo > 0, return signo that signal number in the location referenced by sig.
+* API's covered         :sigwait()
+* Preconditions         :none
+* Preconditions         :none
+* @return               :void
+*/
+
+int recv_pid, snd_pid;
```

Comment:
Instead of global variables, let's deliver them using arguments of task create.

Reply:
modify